src/formula/callable_objects.cpp

```diff
@@ -688,6 +704,9 @@ void team_callable::get_inputs(formula_input_vector& inputs) const
 variant team_callable::get_value(const std::string& key) const
 {
 	if(key == "side") {
+		deprecated_message("team.side", DEP_LEVEL::FOR_REMOVAL, version_info("1.17"), "Use side_number instead.");
+		return variant(team_.side());
+	} else if(key == "side_number") {
```

Comment:
Calling it `side` seems more consistent with SideWML.

Reply:
I deprecated this one because I think it's confusing if it's called "side_number" in one place but "side" in another. I could lower the level on this one, though.